deploy/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.9"
+services:
+  neo4j-db:
+    image: neo4j:4.2-enterprise
+    restart: always
+    command: /mnt/amos/load-dump.sh
+    expose:
+      - "7687"
+    working_dir: /mnt/amos
+    volumes:
+      - ./database/mount:/mnt/amos
+    environment:
+      - NEO4J_ACCEPT_LICENSE_AGREEMENT=yes
+      - DB_PASSWORD=amos
+      - DB_PATH=/mnt/amos/dumps/northwind.dump
+      - NEO4JLABS_PLUGINS=["apoc", "graph-data-science"]
+      - NEO4J_dbms_security_procedures_unrestricted=apoc.*,gds.*
+      - NEO4J_dbms_security_procedures_allowlist=apoc.*,gds.*
+      - NEO4J_apoc_import_file_enabled=true
+    networks:
+      - kmap-network
+  kmap.backend:
+    image: kmap.backend
+    restart: always
+    expose:
+      - "8080"
+    environment:
```

Comment:
We should use an `.env` file here https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option.
It matches the development environment better and adds security since we can use a `.env` file that is located only on the production server, and we could use a different on for integration tests or similar.

Reply:
Done!